cmd/eulabia-sensor/main.go

```diff
@@ -12,12 +12,13 @@ import (
 	"github.com/greenbone/eulabeia/connection"
 	"github.com/greenbone/eulabeia/connection/mqtt"
 	"github.com/greenbone/eulabeia/messages"
+	"github.com/greenbone/eulabeia/messages/cmds"
 	"github.com/greenbone/eulabeia/messages/handler"
 	"github.com/greenbone/eulabeia/sensor/memory"
 )
 
 func main() {
-	topic := "greenbone.sensor"
+	topic := "eulabeia/+/+/sensor"
```

Comment:
Is this a placeholder for a later implementation? Or why are you using a `/+/+/` for a topic?

Reply:
+ means whatever the name is for that subpath.